lib/version_utils.pm

```diff
@@ -257,6 +258,21 @@ sub is_sle_micro {
     return check_version($query, $version, qr/\d{1,}\.\d/);
 }
 
+=head2 is_alp
+
+Check if distribution is ALP
+=cut
+sub is_alp {
+    my $query = shift;
+    my $version = shift // get_var('VERSION');
```

Comment:
```suggestion
    my ($query, $version) = @_;
    $version //= get_var('VERSION');
```

Reply:
Yes, that's probably how I would do it, but I wanted to keep the same style of the other similar methods..